tests/foreman/ui/test_settings.py

```diff
@@ -175,6 +175,47 @@ def test_negative_validate_error_message(session, set_original_property_value):
                 assert is_valid_error_message(str(context.value))
 
 
+@tier2
+def test_positive_input_textarea_large(session):
+    """"Add values to large text box in setting
+
+    :id: 529ddd3a-1271-4043-9006-eac436b08b11
+
+    :expectedresults: Successfully add value to large text box
+
+    :CaseImportance: High
+    """
+    input = f'[ {gen_string("alpha")} ]'
+    property_name = 'host_dmi_uuid_duplicates'
+    with session:
+        session.settings.update(
+            'name = {}'.format(property_name),
+            input
+        )
+        result = session.settings.read('name = {}'.format(property_name))
+        assert result['table'][0]['Value'] == input
+
+
+@tier2
+def test_positive_input_box(session):
+    """
+    :id: 81053b02-0219-4024-bb85-0312dff68fb1
+
+    :expectedresults:  Successfully add value to small input text box
+
+    :CaseImportance: High
+    """
+    input = gen_string('alpha')
+    property_name = 'register_hostname_fact'
+    with session:
+        session.settings.update(
+            'name = {}'.format(property_name),
+            f'{input}'
+        )
+        result = session.settings.read('name = {}'.format(property_name))
+        assert result['table'][0]['Value'] == input
+
+
```

Comment:
What we are testing here? the UI input fields (textbox) or update crud? I hope that is already tested in earlier tests. Do you have any Bugzilla associated with these tests? 

As per my first instinct, we should avoid such tests (with reference to this https://github.com/SatelliteQE/robottelo/issues/6408). If you still want to add these tests please add it in a single test.

Reply:
We are testing the UI input field.  I've added the selector tests as one test.